src/ui/chat/ChatMessageBar.js

```diff
@@ -22,6 +23,13 @@ export default class ChatMessageBar extends Component {
     }
   }
 
+  _onChange(event: SyntheticEvent) {
```

Comment:
The convention we're using in Shin is to use class property arrow functions for event handlers:

`_onChange = (event: SyntheticEvent) => {`

This convention sets `this` appropriately, removing the need for `bind` later on.